pkg/apis/istio/v1beta1/defaults.go

```diff
@@ -147,6 +147,9 @@ func SetDefaults(config *Istio) {
 	if config.Spec.Pilot.EnableProtocolSniffingInbound == nil {
 		config.Spec.Pilot.EnableProtocolSniffingInbound = util.BoolPointer(false)
 	}
+	if config.Spec.Pilot.AdditionalContainerArgs == nil {
+		config.Spec.Pilot.AdditionalContainerArgs = []string{}
```

Comment:
These defaults seems unnecessary, the code will handle `nil` just fine.

Reply:
Hello, done :)